File: combine.py
def combine(firstWord, firstLength, secondWord, secondLength):
    firstWord = firstWord[ :firstLength]
    secondLength = len(secondWord) - secondLength
    secondWord = secondWord[secondLength: ]
    finalWord = firstWord + secondWord
    return finalWord

def main():
    print("Time to go and make a portmanteau!")
    oneWord = input("First word: ")
    oneLength = int(input("Length of first slice: "))
    twoWord = input("Second word: ")
    twoLength = int(input("Length of the second slice: "))
    print(combine(oneWord, oneLength, twoWord, twoLength))

if __name__ == '__main__':
    main()

File: combine.js
var firstWord = prompt("What is the first word? ");
var firstLength = Number(prompt("What is the length you want to cut it at? "));
var secondWord = prompt("What is the second word? ");
var secondLength = Number(prompt("What is the length you want to cut it at? "));
var finalWord;

secondLength = secondWord.length - secondLength;
firstWord = firstWord.slice(0, firstLength);
secondWord = secondWord.slice(secondLength,);
finalWord = firstWord + secondWord;
document.getElementById("demo").innerHTML = finalWord;
